.storybook/main.cjs

```diff
@@ -124,7 +124,6 @@ module.exports = {
     );
 
     // Ensure SVGR is the only loader used for files with .svg extension.
-    // TODO: Figure out why this is needed; only the first-matching rule ought to be used.
```

Comment:
@samwhale So was my assumption incorrect that webpack uses only the first-matching rule?

Reply:
Oh sorry! I thought I added this line in, so I thought we didn't need it anymore.

webpack was matching all of the svg rules, so I grouped the svg rules together in a `oneOf` block: https://github.com/GoogleForCreators/web-stories-wp/pull/5792/files#diff-b724a559a3d863e949957e731407f96e486b45139afbdee10bc1082f5bd229e3R112-R117